### PR TITLE
chore: use singular style in middleware config class name

### PR DESCRIPTION
**Pull request by bowenliang123**
# Description

- as the follow-up for #5489 , apply singular stylein middleware config classes

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] TODO

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods
- [ ] `optional` I have made corresponding changes to the documentation 
- [ ] `optional` I have added tests that prove my fix is effective or that my feature works
- [ ] `optional` New and existing unit tests pass locally with my changes

### CODE REVIEW

File: api/configs/middleware/__init__.py [hunk 1]
```diff
@@ -8,20 +8,20 @@
...
-from configs.middleware.vdb.pgvectors_configs import PGVectoRSConfigs
-from configs.middleware.vdb.qdrant_configs import QdrantConfigs
-from configs.middleware.vdb.relyt_configs import RelytConfigs
-from configs.middleware.vdb.tencent_vector_configs import TencentVectorDBConfigs
-from configs.middleware.vdb.tidb_vector_configs import TiDBVectorConfigs
-from configs.middleware.vdb.weaviate_configs import WeaviateConfigs
-
-
-class StorageConfigs(BaseModel):
+from configs.middleware.vdb.chroma_configs import ChromaConfig
```
Comment (takatost) [inline api/configs/middleware/__init__.py]: The Python filename seems like it can also remove the 's'.

**Review by takatost — CHANGES_REQUESTED** [review]

File: api/configs/middleware/__init__.py (hunk 1 above)
Comment (bowenliang123) [reply api/configs/middleware/__init__.py]: Thanks for the reminder. Fixed.

Comment (bowenliang123) [reply api/configs/middleware/__init__.py]: Thanks for the reminder. Fixed.

**Review by takatost — APPROVED** [review]
LGTM

**Pull request merged**